.gitignore

```diff
@@ -9,6 +9,9 @@ packages/sample-esbuild-app
 packages/sample-view
 packages/sample-package
 packages/nested
+packages/sample-library-package
+packages/sample-depending-package
+packages/sample-library-renamed-package
```

Comment:
let's add a scoped and nested example - which will be the majority of apps for our teams

Reply:
Now that we don't actually rename directories, do we still need this? Wouldn't like to pollute the repo too much.